rust/opendp_tooling/src/codegen/test.rs

```diff
@@ -0,0 +1,36 @@
+use crate::codegen::python::generate_function;
+use crate::{Argument, Function};
+use std::collections::HashMap;
+
+#[test]
+fn test_python_code_generation() {
+    // TODO: What is the least we can fill in below to get it to run?
+    let argument = Argument {
+        name: None,
+        c_type: None,
+        rust_type: None,
```

Comment:
Write a rust function in a string, then use macro quote for a token stream than can be passed to bootstrap. (Usually called by compiler.) ... Or not, because it writes to the file system.

perhaps write a test for `from_ast`?

Reply:
"f64"